tools/ci_connector_ops/ci_connector_ops/qa_checks.py

```diff
@@ -93,9 +105,19 @@ def check_connector_https_url_only(connector: Connector) -> bool:
     Returns:
         bool: Wether the connector code contains only https url.
     """
-    # TODO implement
+    files_with_http_url = set()
+    for filename, line in read_all_files_in_directory(connector.code_directory, path_patterns_to_ignore):
+        if "http://json-schema.org/draft-07/schema#" in line:
```

Comment:
Ha! JSONschema is unsafe.  Maybe 

```suggestion
        if "http://json-schema.org/" in line:
```